lib/ansible/plugins/filter/mathstuff.py

```diff
@@ -134,6 +137,12 @@ def filters(self):
             'symmetric_difference': symmetric_difference,
             'union': union,
 
+            # combinatorial
+            'permutations': itertools.permutations,
+            'combinations': itertools.combinations,
+            'zip': itertools.izip,
```

Comment:
On python 3, `izip` is just `zip`. This is causing the unit test and integration test failures.